.changeset/stupid-rockets-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Added retry delay with exponential backoff to the realtime service.
```

Comment:
I actually don't think we need a changeset here, because the retry behavior is not yet documented / part of the public API.